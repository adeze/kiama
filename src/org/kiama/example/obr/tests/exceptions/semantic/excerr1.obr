(* Exception-related errors in declarations *)

PROGRAM ExcErr1 (i : INTEGER) : INTEGER;

CONST
    (* Two exception vars with same name *)
    FallOverALot : EXCEPTION;
    FallOverALot : EXCEPTION;

    (* Exception var with same name as ordinary var *)
    Bob : EXCEPTION;

VAR
    Bob : INTEGER;

BEGIN
END ExcErr1.
