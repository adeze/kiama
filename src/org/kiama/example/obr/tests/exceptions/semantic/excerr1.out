8.5: FallOverALot is declared more than once
14.5: Bob is declared more than once
